cumulusci.yml

```diff
@@ -37,6 +37,27 @@ tasks:
         options:
             path: unpackaged/config/dev_delete
 
+    deploy_encryption_permset:
+        group: 'Shield Platform Encryption'
+        description: Deploys "Manage Encryption Keys" permission set.
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unpackaged/config/spe
+
+    assign_encryption_permset:
+        group: 'Shield Platform Encryption'
+        description: Assigns the Encryption permission set to the current user for Shield Platform Encryption testing.
+        class_path: cumulusci.tasks.sfdx.SFDXOrgTask
+        options:
+            command: 'force:user:permset:assign -n Encryption'
+    
+    create_tenant_secret:
+        group: 'Shield Platform Encryption'
+        description: Creates a new tenant secret for Shield Platform Encryption testing.
+        class_path: cumulusci.tasks.sfdx.SFDXOrgTask
+        options:
+            command: 'force:data:record:create -s TenantSecret -v "Description=Testing"'
```

Comment:
Got this from https://github.com/wadewegner/sfdx-platformencryption -- the values option was required and for some reason I don't even know where the Description shows up in the UI as I wasn't able to see it on the key record...